eclide/RepositoryFrame.h

```diff
@@ -60,8 +60,6 @@ class CRepositoryFrameT :	public CRepositorySlotImpl,
     {
         T * pT = static_cast<T*>(this);
         ContextState state;
-        if (s.mods.size() + s.attrs.size() == 0)
-            return;
```

Comment:
@GordonSmith I will triple check it.  This code was what I added erroneously and it turns out it was working fine. I just had the idea to totally block it from coming up which turns out to be a BAD idea. Again, I'm triple checking.